assets/scss/default.scss

```diff
@@ -1,5 +1,9 @@
+html {
+  background-color: $white;
```

Comment:
Now all pages have a white background. Is that intended?

Reply:
design said only exhibitions pages should be white, this may be difficult to style

Reply:
I've fixed this